Run-SystemTest.ps1

```diff
@@ -0,0 +1,37 @@
+#Requires -Version 5.0
+
+Begin {
+    $ErrorActionPreference = "stop"
+}
+
+Process {
+    function Exec([scriptblock]$Command) {
+        & $Command
+        if ($LASTEXITCODE -ne 0) {
+            throw ("An error occurred while executing command: {0}" -f $Command)
+        }
+    }
+    
+    $workingDir = Join-Path $PSScriptRoot "src"
+    $outputDir = Join-Path $PSScriptRoot ".output"
+    $sysTestDir = Join-Path $PSScriptRoot "src/system-test/WebApi.MsBuild.SystemTest"
+
+    try {
+        Push-Location $workingDir
+        Remove-Item $outputDir -Force -Recurse -ErrorAction SilentlyContinue
```

Comment:
Is there already something there? If yes maybe we don't to remove stuff we don't own?

Reply:
I guess the purpose is to `clean` the directory from previous step. (Which utilizes the same directory)